opentelemetry-collector/README.md

```diff
@@ -32,9 +32,16 @@ Example Playbook
 
 Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
 
-    - hosts: servers
-      roles:
-         - { role: opentelemetry-collector }
+```
+---
+- name: Install and configure Open Telemetry collector
+  hosts: all
+  become: true
+  tasks:
+    - name: "Include opentelemetry_collector"
+      include_role:
+        name: "opentelemetry-collector"
+```
```

Comment:
nice!
Will you add another one for containerized one?
```
- name: Install and configure Open Telemetry collector in a Docker container
  hosts: all
  become: true
  tasks:
    - name: create inventory
      vars:
        collector_is_containerized: true
      include_role:
        name: opentelemetry_collector
```
🙏

Reply:
Sure, done! :)